spylib/auth_helpers.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from .exceptions import APIException
+from .settings import AUTH_BASE_URL
+from six.moves.urllib.parse import urljoin
+import requests
+
+
+def create_auth_perm(access_token, service_uuid, permission_name, description):
+    headers = {"Authorization": "Bearer %s" % access_token}
+
+    payload = {
+        "owner": service_uuid,
+        "permission_name": permission_name,
+        "description": description,
+    }
+
+    resp = requests.post(
+        urljoin(AUTH_BASE_URL, "api/v1/permissions"),
+        headers=headers,
+        data=payload,
```

Comment:
since this is a `data` attribute, I think you'll need to `json.dumps()` this. Alternatively, you could change `data` to be `json`